tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -269,7 +269,11 @@ static uint8_t supported_commands(const void *cmd, uint16_t cmd_len,
 		tester_set_bit(rp->data, BTP_GAP_SET_EXTENDED_ADVERTISING);
 	}
 
-	*rsp_len = sizeof(*rp) + 4;
+	if (IS_ENABLED(CONFIG_BT_RPA_TIMEOUT_DYNAMIC)) {
+		tester_set_bit(rp->data, BTP_GAP_SET_RPA_TIMEOUT);
+	}
+
+	*rsp_len = sizeof(*rp) + 5;
```

Comment:
So the `+ 5` here indicates the number of octets used. `BTP_GAP_SET_RPA_TIMEOUT` has the value of 0x30 -> 48, so this should be `+ 7` to support setting bit 48